main.go

```diff
@@ -19,6 +24,7 @@ var fixtures = Fixtures{
 				Hash:                  "f47f238d1734881b17e6000cb9fd01248073c900",
 			},
 		},
+		PRNumber: 14,
```

Comment:
We might want to add some PRs from other repositories at one point. 
Similar to go-git-fixtures that contain tests not only from fixtures repository but from some real repos too.